src/main/java/com/money_account_service/utility/RestTemplateWrapper.java

```diff
@@ -0,0 +1,41 @@
+package com.money_account_service.utility;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.money_account_service.dtos.request.AuthorizeRequestDto;
+import com.money_account_service.dtos.response.AuthorizeResponseDto;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Optional;
+
+@AllArgsConstructor
+public class RestTemplateWrapper {
+
+    private RestTemplate restTemplate;
+
+    public static final String AUTHORIZE_URL = "https://localhost:8081/authorize";
+
+    public Optional<AuthorizeResponseDto> authorizeRequest(AuthorizeRequestDto authorizeRequestDto) {
```

Comment:
I think it could be valuable to extract clients to a standalone classes like:
* `UserServiceClient`
* `AccountServiceClient`
Then for some tests it's easier to sub those.